### ISSUE TITLE
Pedestrain training

### ISSUE THREAD

**Issue by iamjeff7** [detectRecog/PointTrack#3]
Dear Author,
How can I train for pedestrian instead of car. Thank you!

**Comment by qa276390** [comment]
Similar question. @detectRecog will you release the pretrained weights for pedestrian

**Comment by detectRecog** [comment]
For segmentation, you can train on pedestrians by switching class_id to 2 rather than 1 (for cars) in the dataset. The change of code for tracking should also be similar.
There are bugs in the pedestrian model. So we do not plan to provide our trained model for pedestrians. However,  for the convenience of comparisons with PointTrack, we have provided the test set results of PointTrack on cars and pedestrians on the GitHub page.

**Comment by EchoAmor** [comment]
Hi, I want to train on pedestrian too, can u explain more specific? I don't find where to switch class_id, thank you very much! @detectRecog
